-------------------------------------------------------------------------------
Test set: com.wipro.HackathonWave20Team2.controller.UserControllerTest
-------------------------------------------------------------------------------
Tests run: 5, Failures: 3, Errors: 0, Skipped: 0, Time elapsed: 8.072 sec <<< FAILURE! - in com.wipro.HackathonWave20Team2.controller.UserControllerTest
testUpdateUser(com.wipro.HackathonWave20Team2.controller.UserControllerTest)  Time elapsed: 0.042 sec  <<< FAILURE!
org.junit.ComparisonFailure: expected:<[User Updated successfully]> but was:<[{"timestamp":1508141276928,"status":404,"error":"Not Found","message":"No message available","path":"/users/2"}]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.wipro.HackathonWave20Team2.controller.UserControllerTest.testUpdateUser(UserControllerTest.java:86)

testGetUser(com.wipro.HackathonWave20Team2.controller.UserControllerTest)  Time elapsed: 0.021 sec  <<< FAILURE!
org.junit.ComparisonFailure: expected:<[User Getting Successfully]> but was:<[{"timestamp":1508141276966,"status":404,"error":"Not Found","message":"No message available","path":"/users/1"}]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.wipro.HackathonWave20Team2.controller.UserControllerTest.testGetUser(UserControllerTest.java:75)

testDeleteUser(com.wipro.HackathonWave20Team2.controller.UserControllerTest)  Time elapsed: 0.026 sec  <<< FAILURE!
org.junit.ComparisonFailure: expected:<[User Deleted successfully]> but was:<[{"timestamp":1508141276993,"status":404,"error":"Not Found","message":"No message available","path":"/users/1"}]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.wipro.HackathonWave20Team2.controller.UserControllerTest.testDeleteUser(UserControllerTest.java:98)

